fandogh_cli/info_collector.py

```diff
@@ -0,0 +1,27 @@
+import platform
+
+import click
+
+from fandogh_cli import VERSION
+from fandogh_cli.utils import format_text, TextStyle
+from fandogh_cli.fandogh_client.error_report_client import report
+
+_static_info = {
+    'python_version': platform.sys.version,
+    'platform': platform.platform(),
+    'cli_version': VERSION
+}
+
+
+def collect(cmd, ctx, exception=None):
```

Comment:
we can consider doing this inside another thread , it would make the CLI more responsive.

Reply:
not really. as CLI process stops on every command. so we cannot guarantee the request has sent to the server before app termination unless we use non daemon thread